### ISSUE TITLE
Can't be used in an addon

### ISSUE THREAD

**Issue by lougreenwood** [crashco/ember-template-component-import#13]
The new `podModulePrefix` assertion added 1.1.1 seems to be too strict and breaks use of this addon in an addon.

Now any attempt to use ember-cli in an addon which depends on ^1.1.1 of the addon fails and throws the assertion:

```
ember-template-component-import: 'podModulePrefix' has not been defined in config/environment.js
```

My current solution is to downgrade to v1.1.0

**Comment by davewasmer** [comment]
@lougreenwood can you verify exactly which version you have installed? Is it 1.1.1 or 1.1.2?

I'm using 1.1.2 just fine via an addon. Are you sure `podModulePrefix` is defined?

**Comment by lougreenwood** [comment]
Yep, I'm on 1.1.2 - although I'm not actually using Pods in the addon and only added the `podModulePrefix: "pods",` to the addon environment.js (although now I think of it, that config may be for the addon dummy).

Sounds like this is a side effect of me using this in a classic addon... but in any case 1.1.0 works fine in an addon :)

**Comment by lifeart** [comment]
this also should be fixed in https://github.com/crashco/ember-template-component-import/pull/12

https://github.com/crashco/ember-template-component-import/pull/12/files#diff-168726dbe96b3ce427e7fedce31bb0bcR101

**Comment by knownasilya** [comment]
@lougreenwood could you test with v1.3.0?
